docs/content/sftp.md

```diff
@@ -128,6 +128,17 @@ Using an ssh-agent is the only way to load encrypted OpenSSH keys at the moment.
 If you set the `--sftp-ask-password` option, rclone will prompt for a
 password when needed and no password has been configured.
 
+If you have a certificate then you can provide the path to the public key that contains the certificate.  For example:
+
+```
+[remote]
+type = sftp
+host = example.com
+user = sftpuser
+key_file = ~/id_rsa
+pubkey_file = ~/id_rsa-cert.pub
+````
+
```

Comment:
Perhaps mention if you have a concatenated private key + cert, then mention it twice at `key_file` and `pubkey_file` - I think that is right isn't it?

Reply:
Comment and example added.